it/connectors/IncomeTaxUserDataConnectorSpec.scala

```diff
@@ -57,6 +58,18 @@ class IncomeTaxUserDataConnectorSpec extends IntegrationTest{
     }
 
     "Return an error result" when {
+
+      "submission returns a 500 when the call is external as the headers won't be passed along" in {
```

Comment:
This test seems to rely on wiremock not being able to match the request due to missing headers, resulting in a 404, which is then handled and converted in an error with InternalServerError

The way the test is written now, the wording should probably be, "Return 500 error when submission returns 404", which I don't think is the intent here.

Reply:
Updated the test name on SASS-687 & merged into here